testdata/taskrouter_responses.go

```diff
@@ -84,3 +84,79 @@ var TaskQueueCreateResponse = []byte(`
     "workspace_sid": "WS7a2aa7d8acc191786ad3c647c5fc3110"
   }
 `)
+
+var WorkflowResponse = []byte(`
+{
+    "account_sid": "AC58f1e8f2b1c6b88ca90a012a4be0c279",
+    "assignment_callback_url": "http://example.com",
+    "configuration": "task-routing:\n  - filter: \n      - 1 == 1\n    target:\n      - queue: WQ63868a235fc1cf3987e6a2b67346273f\n        set-priority: 0\n",
```

Comment:
Also returning JSON-in-JSON here, the YAML is a mistake.

```
    "configuration": "{\"task_routing\":{\"default_filter\":{\"queue\":\"WQ0c1274082082355320d8a41f94eb57aa\"}}}",
```

Reply:
Updated 👍